src/pods/edit-table/edit-table.business.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { removeField } from './edit-table.business';
+import { TableVm } from './edit-table.vm';
+import { GUID } from '@/core/model';
+
+const table: TableVm = {
+  id: '1',
+  fields: [
+    {
+      id: '1',
+      PK: true,
+      FK: false,
+      name: 'field1',
+      type: 'string',
+      children: [],
+      isCollapsed: false,
+    },
+    {
+      id: '2',
+      PK: true,
+      FK: false,
+      name: 'field2',
+      type: 'string',
+      children: [
+        {
+          id: '3',
+          PK: true,
+          FK: false,
+          name: 'child1 field2',
+          type: 'string',
+          children: [],
+          isCollapsed: false,
+        },
+      ],
+      isCollapsed: false,
+    },
+  ],
+
+  tableName: 'table1',
+  x: 0,
+  y: 0,
+};
+
+describe('edit table - removeField function', () => {
```

Comment:
Add an edge case, what happens if you try to remove a field Id that doesn't exists, in this case it should return exactly the same colletion

Reply:
Could that be included in the last case?